lib/teams/index.ts

```diff
@@ -1,11 +1,15 @@
-import { KubernetesManifest, ServiceAccount } from 'aws-cdk-lib/aws-eks';
+import { KubernetesManifest, ServiceAccount, Cluster } from 'aws-cdk-lib/aws-eks';
 import * as iam from 'aws-cdk-lib/aws-iam';
-import { IRole } from 'aws-cdk-lib/aws-iam';
-import { CfnOutput } from 'aws-cdk-lib';
+import { CfnOutput, Aws, CfnJson,  } from 'aws-cdk-lib';
 import { CsiSecretProps, SecretProviderClass } from '../addons/secrets-store/csi-driver-provider-aws-secrets';
 import { ClusterInfo, Team, Values } from '../spi';
-import { applyYamlFromDir } from '../utils/yaml-utils';
+import { applyYamlFromDir, readYamlDocument, loadYaml } from '../utils/yaml-utils';
 import { DefaultTeamRoles } from './default-team-roles';
+import { FederatedPrincipal, IManagedPolicy, IRole, ManagedPolicy, PolicyStatement, Role } from "aws-cdk-lib/aws-iam";
+import * as SimpleBase from 'simple-base';
```

Comment:
Minor: please use lowercase for import alias. E.g. consistent with `import * as cdk` or `import * as ec2`, so simplebase. Applies to `NamespaceUtils`-> nsutils